spec/hydra/works/models/concerns/file_set/virus_check_spec.rb

```diff
@@ -50,4 +51,15 @@ def scanfile(path)
       expect(subject.detect_viruses).to eq true
     end
   end
+
+  # TODO: Write test that does not depend on directly calling private methods.
+  # Testing private methods like this is ill advised.
```

Comment:
I think the way to avoid calling #local_path_for_file directly is to call #detect_viruses with original_file being a stub that responds to #read but not #path.